vital/types/detected_object.h

```diff
@@ -43,10 +43,11 @@
 #include <vector>
 
 #include <vital/types/detected_object_type.h>
-#include <vital/types/vector.h>
 #include <vital/types/bounding_box.h>
 #include <vital/types/descriptor.h>
+#include <vital/types/geo_point.h>
 #include <vital/types/image_container.h>
+#include <vital/types/vector.h>
```

Comment:
[[includes.order]](../tree/master/doc/code-style.rst) ...thanks for fixing! :+1: